app/src/main/java/woowacourse/movie/MovieDetailActivity.kt

```diff
@@ -0,0 +1,156 @@
+
+        initView(movieData)
+        setClickListener(movieData)
+        setSpinnerSelectedListener(movieSchedule, scheduleDate, savedInstanceState)
+        setSpinnerAdapter(scheduleDate, movieSchedule)
+        reloadTicketCountInstance(savedInstanceState)
+    }
+
+    private fun setSpinnerAdapter(scheduleDate: List<String>, movieSchedule: MovieSchedule) {
+        dateSpinner.adapter = ArrayAdapter(
+            this,
+            android.R.layout.simple_spinner_item,
+            scheduleDate
+        )
+        timeSpinner.adapter = ArrayAdapter(
+            this,
+            android.R.layout.simple_spinner_item,
+            movieSchedule.getScheduleTimes(scheduleDate.first())
+        )
+    }
+
+    private fun initView(movieData: Movie) {
+        findViewById<ImageView>(R.id.iv_movie_poster).setImageResource(movieData.poster)
+        findViewById<TextView>(R.id.tv_movie_title).text = movieData.title
+        findViewById<TextView>(R.id.tv_movie_release_date).text = movieData.releaseDate
+        findViewById<TextView>(R.id.tv_movie_running_time).text = movieData.runningTime
+        findViewById<TextView>(R.id.tv_movie_synopsis).text = movieData.synopsis
+    }
+
+    private fun reloadTicketCountInstance(savedInstanceState: Bundle?) {
+        if (savedInstanceState != null) {
+            personCountTextView.text = savedInstanceState.getString(TICKET_COUNT_KEY)
+        }
+    }
+
+    private fun processMovieNullData(movieData: Movie) {
+        if (movieData == Movie.emptyData) {
+            Toast.makeText(this, getString(R.string.cant_get_movie_data), Toast.LENGTH_SHORT).show()
+            this.finish()
+        }
+    }
+
+    private fun setSpinnerSelectedListener(
+        movieSchedule: MovieSchedule,
+        scheduleDate: List<String>,
+        savedInstanceState: Bundle?
+    ) {
+        dateSpinner.onItemSelectedListener = object : AdapterView.OnItemSelectedListener {
+            override fun onItemSelected(
+                adapterView: AdapterView<*>?,
+                view: View?,
+                position: Int,
+                id: Long
+            ) {
+                timeSpinner.adapter = ArrayAdapter(
+                    this@MovieDetailActivity,
+                    android.R.layout.simple_spinner_item,
+                    movieSchedule.getScheduleTimes(scheduleDate[position])
+                )
+                if (needSpinnerInitialize && savedInstanceState != null) {
+                    timeSpinner.setSelection(
+                        (
+                            savedInstanceState.getString(TIME_KEY)
+                                ?: movieSchedule.getScheduleTimes(dateSpinner.selectedItem.toString())
+                                    .first()
+                            ).toInt()
+                    )
+                    needSpinnerInitialize = false
+                }
+            }
+
+            override fun onNothingSelected(parent: AdapterView<*>?) {}
```

Comment:
```suggestion
            override fun onNothingSelected(parent: AdapterView<*>?) = Unit
```
구현부가 없는 경우에는 Unit을 활용하면 좀 더 코드를 깔끔하게 만들 수 있어요